gordon/main.py

```diff
@@ -99,8 +99,13 @@ def setup(config_root=''):
     log_level = logging_config.get('level', 'INFO').upper()
     log_handlers = logging_config.get('handlers') or ['syslog']
```

Comment:
You could do the `pop`s already up here and save a couple lines.

Reply:
I thought about that, but I don't want to globally edit config. IIRC the metrics logger uses the level up here if it isn't set in its own config.

Reply:
Sorry, I meant you could do the `copy` up here too, as part of line 98 maybe?  But if you think it makes things more confusing, I have no problem skipping it.

Reply:
ahhh I get what you're saying; yeah that'd look nicer, good idea.

Reply:
hm I don't think I can quickly fix this up because of the ability to set a default in `.get` 🤔

Reply:
I think `.pop` takes a default as well.

Reply:
wat. I tried earlier and it wasn't behaving. (╯°□°）╯︵ ┻━┻